app/model/schema/token.py

```diff
@@ -49,6 +49,18 @@ class CreateTokenHoldersCollectionRequest(BaseModel):
 @dataclass
 class ListAllTokenHoldersQuery:
     exclude_owner: Annotated[Optional[bool], Query(description="exclude owner")] = False
+    offset: Annotated[Optional[int], Query(description="start position", ge=0)] = None
+    limit: Annotated[Optional[int], Query(description="number of set", ge=0)] = None
+
+
+class SearchTokenHoldersRequest(BaseModel):
+    account_address_list: list[StrictStr] = Field(
+        [],
+        description="list of token address (**this affects total number**)",
+    )
```

Comment:
Is the reason why you are not validating using `ValidatedEthereumAddress`` considering performance?

Reply:
The reason for this is to avoid strict validation in the GET request query.
I am wondering if I should use `ValidatedEthereumAddress` for GET schema.

Reply:
OK, I understand that since it is a search query, the user is allowed to enter any string.